Revolg DOCSUP-1000 add max server memory usage setting (#125)
BayoNet/ClickHouse: docs/en/operations/settings/query-complexity.md

See also the description of [max\_memory\_usage](#settings_max_memory_usage).

> ## max\_server\_memory\_usage {#max_server_memory_usage}

Reviewer: ```suggestion
## max_server_memory_usage {#max_server_memory_usage}
```

> Limits total memory usage of the server. Can be specified only for default profile.

Reviewer: ```suggestion
Limits total RAM usage by the ClickHouse server. You can specify it only for the default profile.
```

Possible values:

-   Positive integer.
-   0 — Unlimited.

Default value: `0`.

**Additional Info**

> Default values are defined in [Settings.h](https://github.com/ClickHouse/ClickHouse/blob/master/src/Core/Settings.h#L289). 

Reviewer: Delete this sentence.

> On servers with very low memory amount and swap you may have to add `max_server_memory_usage_to_ram_ratio` in `config.xml` and set it to a value larger than `1`.

Reviewer: ```suggestion
On hosts with low RAM and swap, you possibly need setting `max_server_memory_usage_to_ram_ratio > 1`.
```

**See also**

> -   [max\_memory\_usage](#settings_max_memory_usage)

Reviewer: ```suggestion
-   [max_memory_usage](#settings_max_memory_usage)
```
